### PR TITLE
i18n: some CN translations for endwalker

### PR DESCRIPTION
**Pull request by DieMoe233**

### CODE REVIEW

File: ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts [hunk 1]
```diff
@@ -79,6 +80,7 @@ const triggerSet: TriggerSet<Data> = {
           de: 'Hinter den ${num}. Orb',
           fr: 'Allez derrière l\'orbe ${num}',
           ko: '${num}번 구슬 뒤로',
+          cn: '去第${num}个球的终点方向贴边',
```
Comment (MaikoTan) [inline ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts:83]:
```suggestion
          cn: '去第${num}个球的终点方向贴边',
          ko: '${num}번 구슬 뒤로',
```

File: ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts [hunk 2]
```diff
@@ -134,6 +137,7 @@ const triggerSet: TriggerSet<Data> = {
           de: 'Geh hinter den ruhigen Klon',
           fr: 'Allez derrière le vrai clone',
           ko: '가만히 있는 분신 뒤로',
+          cn: '找不动的boss',
```
Comment (MaikoTan) [inline ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts:140]:
```suggestion
          cn: '找不动的boss',
          ko: '가만히 있는 분신 뒤로',
```

File: ui/raidboss/data/06-ew/trial/zodiark.ts [hunk 3]
```diff
@@ -42,6 +42,7 @@ const triggerSet: TriggerSet<Data> = {
           de: 'Geh nach NO (Rückstoß)',
           fr: 'Allez au NE (poussée)',
           ko: '북동쪽으로 (넉백)',
+          cn: '去右上(东北)',
```
Comment (MaikoTan) [inline ui/raidboss/data/06-ew/trial/zodiark.ts:45]:
```suggestion
          cn: '去右上(东北)',
          ko: '북동쪽으로 (넉백)',
```

**Review by MaikoTan — CHANGES_REQUESTED** [review]

**Comment by quisquous** [comment]
Sorry, would you mind also fixing the conflicts?

**Comment by DieMoe233** [comment]
> Sorry, would you mind also fixing the conflicts?

fixed.

**Pull request merged**